app/shared/language/language-mapper.ts

```diff
@@ -0,0 +1,14 @@
+import { availableLanguages, LanguageCode, OTHER_LANGUAGE_SCHEMA } from './available-languages';
+
+export class LanguageMapper {
+  private static getLanguageByCode(code: string, languageCode: LanguageCode) {
+    return availableLanguages.find(item => code === item[languageCode]);
+  }
+
+  static fromTo(code: string, from: LanguageCode, to: LanguageCode) {
```

Comment:
I'm not really sold on this mapper.  `LanguageMapper.fromTo('EN', 'ISO639_1', 'ISO639_3')` reads weird.

What if this is just a "getter" and you use whichever key you want after the get?  For example: 
`LanguageUtils.get('EN', 'ISO639_1).ISO639_3`

What is the advantage of the mapper? simply the default value?  Maybe that can be included into the getter as well?

Let's discuss this on the call tomorrow?

Reply:
Yes, we can I discuss this, but the main downside of this is if we change this we're going to need to change all clients consuming this new style on the codebase